### ISSUE TITLE
[CI] Code coverage upload failure

### ISSUE THREAD

**Issue by dlangBugzillaToGithub** [dlang/dmd#20445]
### Description

````markdown
The Github Actions coverage CI checks for dmd consistently fail with:

> [2024-04-22T12:43:32.048Z] ['error'] There was an error running the uploader: Error uploading to https://codecov.io: Error: There was an error fetching the storage URL during POST: 404 - {'detail': ErrorDetail(string='Unable to locate build via Github Actions API. Please upload with the Codecov repository upload token to resolve issue.', code='not_found')}

See also: https://github.com/dlang/dmd/pull/16401#issuecomment-2068179334
````

**Comment by dlangBugzillaToGithub** [comment]
#### dlang-bot commented on 2024-04-22T14:05:44Z



````markdown
@dkorpel created dlang/dmd pull request #16404 "Bugzilla issue 24518 - disable GH action coverage checks" mentioning this issue:

- Bugzilla issue 24518 - disable GH action coverage checks

https://github.com/dlang/dmd/pull/16404
````

**Comment by dlangBugzillaToGithub** [comment]
#### dlang-bot commented on 2024-04-23T07:18:04Z



````markdown
dlang/dmd pull request #16404 "Bugzilla issue 24518 - disable GH action coverage checks" was merged into stable:

- 6f0e6ec563dd34b83a98e2694bd9f9f2dfd4d078 by Dennis Korpel:
  Bugzilla issue 24518 - disable GH action coverage checks

https://github.com/dlang/dmd/pull/16404
````

**Comment by dlangBugzillaToGithub** [comment]
#### dlang-bot commented on 2024-04-23T12:44:50Z



````markdown
@dkorpel created dlang/dmd pull request #16406 "Merge stable" mentioning this issue:

- Bugzilla issue 24518 - disable GH action coverage checks (#16404)

https://github.com/dlang/dmd/pull/16406
````

**Comment by dlangBugzillaToGithub** [comment]
#### dlang-bot commented on 2024-04-23T14:24:19Z



````markdown
dlang/dmd pull request #16406 "Merge stable" was merged into master:

- 111ae7308b4fc1a39fba01bf9df2f4fd8d812701 by Dennis:
  Bugzilla issue 24518 - disable GH action coverage checks (#16404)

https://github.com/dlang/dmd/pull/16406
````

**Comment by thewilsonator** [comment]
@dkorpel was this fixed by #16404 ?
